### PR TITLE
Disc 235/portalpage setup

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Portal frontpage.

### CODE REVIEW

**Review by jorntx — COMMENTED** [review]
Very nice 'portal page' and refactor you did here. I only have small comments and suggestions and of course that 'sticky bug' we talked about.

And I got to say the new frontpage really does some good to the apps look and feel - good job! 

Review the comments and then we will merge

File: src/assets/global-styles.css
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/assets/global-styles.css:1]: Yea, i thought i needed to use it for something. I didn't need it now - figured we might later. I'll remove it, we can always add it if we actually need it at some point.

File: src/components/GridDisplay.vue [hunk 1]
```diff
@@ -0,0 +1,228 @@
+<template>
+	<div class="grid-display">
+		<h2 v-if="title">{{ title }}</h2>
+		<div :class="draggable ? 'record-grid draggable row-' + rowNr : 'record-grid row-' + rowNr">
+			<div
+				v-bind:key="item"
```
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/components/GridDisplay.vue:6]: Good point. Fixed.

File: src/components/GridDisplay.vue [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+			}
+		},
+		stopMovementOnParent() {
+			if (this.slidingElement) {
+				this.isDown = false;
+			}
+		},
+		calculateMovement(e: MouseEvent) {
+			if (this.slidingElement) {
+				if (!this.isDown) return;
```
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/components/GridDisplay.vue:101]: It's a fair point. Will fix.

File: src/components/search/search-component.ts [hunk 3]
```diff
@@ -106,7 +106,16 @@ const SEARCH_COMPONENT_TEMPLATE = /*html*/ `
...
 	<style>
-	  .material-icons {
+	/* clears the ‘X’ from Internet Explorer */
+		input[type=search]::-ms-clear { display: none; width : 0; height: 0; }
+		input[type=search]::-ms-reveal { display: none; width : 0; height: 0; }
+		/* clears the ‘X’ from Chrome */
+		input[type="search"]::-webkit-search-decoration,
+		input[type="search"]::-webkit-search-cancel-button,
+		input[type="search"]::-webkit-search-results-button,
+		input[type="search"]::-webkit-search-results-decoration { display: none; }
```
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/components/search/search-component.ts:116]: Yea. I was kinda 2-way about it. I'll do another JIRA for adding our own button instead that removes the content in the searchbox *and* resets the search result pack to the portal page. Imho that would be ideal for this.

File: src/views/Search.vue [hunk 4]
```diff
@@ -3,29 +3,74 @@
...
 		</div>
-		<div class="container">
-			<div class="row">
-				<div class="search-resultset">
-					<div class="search-facets">
-						<Facets :facet-results="searchResultStore.facetResult" />
+		<Transition name="fade">
+			<div
+				v-if="searchResultStore.searchResult.length === 0"
+				key="2"
```
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/views/Search.vue:37]: Okey, so, we wanna show the portal stuff whenever there is no search-result. We also want it to have a decent animation in for (at least for now). I had some trouble with this between divs, untill i added keys to the 2 divs, so Vue knew which was which. I later ran into some other difficulties, and changed even more, but apparently forgot to remove the keys. Will remove then now :)

**Review by jorntx — APPROVED** [review]
Looks good!

**Pull request merged**